doc/api/child_process.markdown

```diff
@@ -176,7 +176,7 @@ replace the existing process and uses a shell to execute the command.*
 
 ### child_process.execFile(file[, args][, options][, callback])
 
-* `file` {String} The filename of the program to run
+* `file` {String} A relative or absolute path to an executable file
```

Comment:
That's not right. In my comment I showed the execing "date" works fine, and it not a relative or absolute path. The previous docs are correct, what are you trying to fix?